### PR TITLE
[10.0][MIG] stock disable force availability button

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
Migration done. This https://github.com/OCA/stock-logistics-workflow/pull/326 can be closed. @Cedric-Pigeon

File: stock_disable_force_availability_button/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (Cedric-Pigeon) [inline stock_disable_force_availability_button/__openerp__.py]: this file should disappear

**Comment by rousseldenis** [comment]
Work done, please don't squash last 2 commits

**Review by oihane — APPROVED** [review]
LGTM

**Review by Cedric-Pigeon — APPROVED** [review]
Good to me, thanks

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**